doc/source/NewSiteConfigs.rst

```diff
@@ -226,20 +229,23 @@ Remember to activate the ``lua`` module environment and have MacTeX in your sear
    spack external find --scope system \
        --exclude bison --exclude openssl \
        --exclude python --exclude gettext
-   spack external find --scope system libiconv
```

Comment:
WIth this change, here and line 241 below, do we also need to add `brew install libiconv` to the brew commands listed above (around line 141)?

Reply:
done